packages/core/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.5-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.0-bin.zip
```

Comment:
when updating Gradle it is better to use `./gradlew wrapper --gradle-version 8.0 --distribution-type all` command. Advantages:

* It will also update the wrapper script as well
* By supplying `all` distribution type you will keep the distribution with all the javadoc included, which is handy.

Same comment applies to other Gradle updates in this PR.

Reply:
Sounds good! Thank you